tests/components/deconz/test_binary_sensor.py

```diff
@@ -123,51 +112,38 @@ async def test_allow_clip_sensor(hass):
         options={deconz.gateway.CONF_ALLOW_CLIP_SENSOR: True},
         get_state_response=data,
     )
-    assert "binary_sensor.presence_sensor" in gateway.deconz_ids
-    assert "binary_sensor.temperature_sensor" not in gateway.deconz_ids
-    assert "binary_sensor.clip_presence_sensor" in gateway.deconz_ids
-    assert "binary_sensor.vibration_sensor" in gateway.deconz_ids
-    assert len(hass.states.async_all()) == 4
-
-    presence_sensor = hass.states.get("binary_sensor.presence_sensor")
-    assert presence_sensor.state == "off"
-
-    temperature_sensor = hass.states.get("binary_sensor.temperature_sensor")
-    assert temperature_sensor is None
 
-    clip_presence_sensor = hass.states.get("binary_sensor.clip_presence_sensor")
-    assert clip_presence_sensor.state == "off"
+    assert len(hass.states.async_all()) == 4
+    assert hass.states.get("binary_sensor.presence_sensor").state == "off"
+    assert hass.states.get("binary_sensor.temperature_sensor") is None
+    assert hass.states.get("binary_sensor.clip_presence_sensor").state == "off"
+    assert hass.states.get("binary_sensor.vibration_sensor").state == "on"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)